### PR TITLE
add the commit message style guide

### PR DESCRIPTION
**Pull request by shafreeck**
This documentation constraint the commit message style of Titan, it is part of the contribution guide.

### CODE REVIEW

File: docs/commit-message-style.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+### Available `<scope>`
+
+* command
+* db
+* server
+* resp
+* context
+* conf
+* metric
```
Comment (nioshield) [inline docs/commit-message-style.md]: tools 是否也需要加进去?

Comment (shafreeck) [reply docs/commit-message-style.md]: The scope is optional, so you can use any other scope which not being listed here.

**Comment by jl2005** [comment]
LGTM

File: docs/commit-message-style.md (hunk 1 above)
Comment (shafreeck) [reply docs/commit-message-style.md]: I added a description of the scope not listed here.

**Pull request merged**